extensions/PhabBugz/lib/Util.pm

```diff
@@ -273,19 +273,26 @@ sub add_comment_to_revision {
 
 sub get_project_phid {
     my $project = shift;
+    my $memcache = Bugzilla->memcached;
 
-    my $data = {
-        queryKey => 'all',
-        constraints => {
-            name => $project
-        }
-    };
+    # Check memcache
+    my $project_phid = $memcache->get({ key => "phab_project_phid_" . $project });
```

Comment:
question about this: how long do you expect it to be in the cache? 
also, possibly you want to use get_config() for this key.

Reply:
PHID does not change. What is the policy?

Reply:
As log as possible - PHID does not change.